server/lib/helpers/ApiError.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+const ApiErrorCodes = {
+  serverError: 'SERVER_ERROR'
+}
+
+class ApiError extends Error {
+  constructor (message, {status, code} = {}) {
+    super(message)
+
+    // Set Generic error message
+    this.message = message
```

Comment:
Just jumping in here since you asked me to take a look at your PR. :D

Is this line necessary?  I believe "this.message" is set in super constructor

Reply:
Definitely!